vllm/distributed/device_communicators/cuda_wrapper.py

```diff
@@ -0,0 +1,145 @@
+# This file is a pure Python wrapper for the cudart library.
+# It avoids the need to compile a separate shared library, and is
+# convenient for use when we just need to call a few functions.
```

Comment:
```suggestion
"""
This file is a pure Python wrapper for the cudart library.
It avoids the need to compile a separate shared library, and is
convenient for use when we just need to call a few functions.
"""
```

Reply:
fixed in [9461195](https://github.com/vllm-project/vllm/pull/5528/commits/94611959ceb728e08098f2cbe03dcc6edc5a60fb)